src/sass/abstracts/_variables.scss

```diff
@@ -3,6 +3,17 @@ $color-primary: #a12347;
 $color-secondary: #263454;
 $color-tertiary: #d9dbf1;
 
+$dark-grey: #333447;
+$dark-gray: #333447;
```

Comment:
Maybe we can use actual css variables instead of sass variables? This can allow us to add dark mode or theme changes on the fly later.

Reply:
Yeah, you are right. That was my original intent, but I didn't want to complicate things too much yet. I don't think it would be too hard to go back through and replace them with css variables if we decide to do so.

I just don't want people to get confused about the different variable types yet